salon_creator/lib/app.dart

```diff
@@ -1,19 +1,47 @@
 import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:firebase_auth/firebase_auth.dart';
 import 'package:flutter/material.dart';
-import 'package:salon_creator/constant/constants.dart' as constants;
+import 'package:salon_creator/color.dart';
+import 'package:salon_creator/common/constants.dart' as constants;
 import 'package:salon_creator/screens/home.dart';
-import 'package:salon_creator/screens/login_page.dart';
+import 'package:salon_creator/screens/login_screen.dart';
+import 'package:salon_creator/screens/salon_create_application_screen.dart';
+import 'package:salon_creator/screens/salon_create_screen.dart';
+import 'package:salon_creator/screens/screen.dart';
 
 class SalonCreatorApp extends StatelessWidget {
   const SalonCreatorApp({Key key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
+      theme: ThemeData(
+        fontFamily: 'NotoSansJP',
+        primaryColor: primeColor,
+        textButtonTheme: TextButtonThemeData(
+          style: ButtonStyle(
+            foregroundColor: MaterialStateProperty.all(Colors.white),
+          ),
+        ),
+        dividerColor: Color.fromRGBO(193, 193, 193, 1),
+        appBarTheme: AppBarTheme(
+            shadowColor: Colors.transparent,
+            color: Colors.white,
+            foregroundColor: Colors.black,
+            textTheme: TextTheme(
+              headline6: TextStyle(
+                color: Colors.black,
+                fontSize: 20,
+                fontFamily: 'NotoSansJP',
+              ),
+            )),
+      ),
       routes: {
+        '/salon_creation': (context) => SalonCreationScreen(),
         '/home': (context) => HomePage(),
         '/login': (context) => LoginPage(),
+        '/application': (context) => ApplicationScreen(),
```

Comment:
I think it makes more sense to call this registration
```suggestion
        '/salon_registration': (context) => SalonRegistrationScreen(),
```